docs/jwst/outlier_detection/outlier_detection.rst

```diff
@@ -37,8 +37,8 @@ Specifically, this routine performs the following operations:
     save_intermediate_results: specifies whether or not to save any products
                                created during outlier_detection [default=False]
     resample_data: specifies whether or not to resample the input data [default=True]
-    good_bits: Sum of DQ integer values which should be considered good when
-               creating weight and median images [default=6]
+    bad_bits: Sum of DQ integer values which should be considered bad when
```

Comment:
The purpose of having the flag mnemonics (DO_NOT_USE, JUMP_DET, ...) was so that users wouldn't have to remember numerical values and so that we can make changes to the numerical values without it affecting anything. So it would be better if a param like this (whether it's `good_bits` or `bad_bits`) could be a list of mnemonic names, rather than a numerical value.